docs/source/models.rst

```diff
@@ -2654,6 +2654,77 @@ avoid failed inference requests.
 
 \***** If using `pyfunc` in MLflow Model Serving for realtime inference, the raw audio in bytes format must be base64 encoded prior to submitting to the endpoint. String inputs will be interpreted as uri locations.
 
+Using inference_config and dictionary parameters for `transformers` inference
```

Comment:
I think we should also add docs to https://mlflow.org/docs/latest/models.html#model-signature to explain what `parameters` are, what they're used for, and include some examples.

Reply:
I'm thinking of adding it in a follow-up PR

Reply:
Sounds good :)

Reply:
Updated.